Source/WebCore/loader/cache/CachedResource.cpp

```diff
@@ -394,6 +394,11 @@ bool CachedResource::isCrossOrigin() const
     return m_responseTainting != ResourceResponse::Tainting::Basic;
 }
 
+bool CachedResource::isCORSCrossOrigin() const
+{
+    return m_responseTainting != ResourceResponse::Tainting::Basic && m_responseTainting != ResourceResponse::Tainting::Cors;
```

Comment:
It might be better to align with the spec strictly by checking opaque or opaqueredirect.

Reply:
ok